Project.toml

```diff
@@ -21,7 +22,6 @@ RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 RuntimeGeneratedFunctions = "7e49a35a-f44a-4d26-94aa-eba1b4ca6b47"
 SciMLOperators = "c0aeaf25-5076-4817-a8d5-81caf7dfa961"
-PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
```

Comment:
why remove precompile tools?

Reply:
I don't know how that got removed! added it back